data/kubemacpool/003-deployment.yaml

```diff
@@ -13,6 +13,7 @@ spec:
     matchLabels:
       control-plane: mac-controller-manager
       controller-tools.k8s.io: "1.0"
+      app: kubemacpool
```

Comment:
This change failed CI, please take a look.

Reply:
Maybe you wanted to add it to labels, not matchLabels

Reply:
Yes! Blind me